src/ui/CategoryFacet/CategoryValueRoot.ts

```diff
@@ -0,0 +1,21 @@
+import { CategoryValueParent, CategoryValue } from './CategoryValue';
+import { CategoryFacetTemplates } from './CategoryFacetTemplates';
+import { Dom } from '../../utils/Dom';
+import { CategoryChildrenValueRenderer } from './CategoryValueChildrenRenderer';
+
+export class CategoryValueRoot implements CategoryValueParent {
+  private children: CategoryValue[] = [];
+  public categoryChildrenValueRenderer: CategoryChildrenValueRenderer;
+
+  constructor(element: Dom, categoryFacetTemplates: CategoryFacetTemplates) {
+    this.categoryChildrenValueRenderer = new CategoryChildrenValueRenderer(element, categoryFacetTemplates, this);
+  }
+
+  public getPath(partialPath: string[] = []) {
+    return partialPath;
+  }
+
+  public getChildren() {
```

Comment:
If the `getter` is not doing anything, I think I prefer to simply make the property public.

If we need to change that in the future, we can refactor to create a `public get children` method

Reply:
I did that in order to avoid anyone trying to add a value directly using the array or attempt any manipulation. Adding and removing values from the children requires a re-render, I think I prefer keeping this as a getter and exposing methods to add and remove.

I think I'll use a getter.